src/pdsparsemat.jl

```diff
@@ -52,6 +52,7 @@ end
 
 Base.inv(a::PDSparseMat{T}) where {T<:Real} = PDMat(inv(a.mat))
 LinearAlgebra.logdet(a::PDSparseMat) = logdet(a.chol)
+LinearAlgebra.sqrt(A::PDSparseMat) = PDSparseMat(sparse(sqrt(Matrix(A))))
```

Comment:
I would have assumed that the square root of a sparse matrix is not necessarily sparse? So if we want to wrap it with a `AbstractPDMat` maybe we should rather define
```suggestion
LinearAlgebra.sqrt(A::PDSparseMat) = PDMat(sqrt(Hermitian(Matrix(A))))
```